plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergBigLakeMetastoreConnectorSmokeTest.java

```diff
@@ -0,0 +1,375 @@
+    }
+
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        return switch (connectorBehavior) {
+            case SUPPORTS_CREATE_MATERIALIZED_VIEW,
+                 SUPPORTS_RENAME_MATERIALIZED_VIEW,
+                 SUPPORTS_RENAME_SCHEMA,
+                 SUPPORTS_RENAME_TABLE -> false;
+            default -> super.hasBehavior(connectorBehavior);
+        };
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        Path gcpCredentialsFile = Files.createTempFile("gcp-credentials", ".json");
+        gcpCredentialsFile.toFile().deleteOnExit();
+        Files.write(gcpCredentialsFile, GCS_JSON_KEY_BYTES);
+        String projectId = OBJECT_MAPPER.readTree(GCS_JSON_KEY_BYTES).get("project_id").asText();
+
+        return IcebergQueryRunner.builder(SCHEMA)
+                .addIcebergProperty("iceberg.file-format", format.name())
+                .addIcebergProperty("iceberg.register-table-procedure.enabled", "true")
+                .addIcebergProperty("iceberg.unique-table-location", "false")
+                .addIcebergProperty("iceberg.catalog.type", "rest")
+                .addIcebergProperty("iceberg.rest-catalog.uri", "https://biglake.googleapis.com/iceberg/v1beta/restcatalog")
+                .addIcebergProperty("iceberg.rest-catalog.warehouse", "gs://" + GCP_STORAGE_BUCKET)
+                .addIcebergProperty("iceberg.rest-catalog.security", "GOOGLE")
+                .addIcebergProperty("iceberg.rest-catalog.google-project-id", projectId)
+                .addIcebergProperty("iceberg.rest-catalog.view-endpoints-enabled", "false")
+                .addIcebergProperty("iceberg.writer-sort-buffer-size", "1MB")
+                .addIcebergProperty("iceberg.allowed-extra-properties", "write.metadata.delete-after-commit.enabled,write.metadata.previous-versions-max")
+                .addIcebergProperty("fs.native-gcs.enabled", "true")
+                .addIcebergProperty("gcs.json-key-file-path", gcpCredentialsFile.toString())
+                .setSchemaInitializer(SchemaInitializer.builder()
+                        .withSchemaName(SCHEMA)
+                        .withClonedTpchTables(REQUIRED_TPCH_TABLES)
+                        .withSchemaProperties(ImmutableMap.of("location", "'gs://%s/%s'".formatted(GCP_STORAGE_BUCKET, SCHEMA)))
+                        .build())
+                .build();
+    }
+
+    @AfterAll
+    void cleanup()
+            throws Exception
+    {
+        // Avoid DROP SCHEMA CASCADE since BigLake metastore returns null when loading a table with missing metadata file
+        fileSystem.deleteDirectory(Location.of(schemaPath()));
+        for (Object tableName : computeActual("SHOW TABLES IN " + SCHEMA).getOnlyColumnAsSet()) {
+            dropTableFromMetastore((String) tableName);
+        }
+        getQueryRunner().execute("DROP SCHEMA " + SCHEMA);
```

Comment:
assertUpdate ?

Reply:
This is not a test method. No need to assert anything.